docs-chef-io/content/inspec/cloud_security.md

```diff
@@ -0,0 +1,212 @@
+
+#### Approaches to the AWS Shared Responsibility Model
+
+Amazon’s responsibilities fall into two categories in the AWS shared responsibility model. The first is the hardware, including all Regions, Availability Zones, and Edge Locations of the AWS global infrastructure. The second category is managing the software for the compute, storage, database, and networking aspects of cloud computing.
+
+The customer, meanwhile, controls the data and configurations within that cloud environment. In the AWS shared responsibility model, customer responsibility falls into categories such as:
+
+- Customer data stored in the cloud
+- Platform, application, and identity & access management
+- Operating system, network, and firewall configuration
+- Data encryption, integrity, and authentication
+
+The **AWS Shared Responsibility Model** outlines how AWS commits to their cloud's "security" and where customers need to focus on security "in" the cloud.
+
+As a customer/company managing your own IT infrastructure (either bare metal or VMs) need to create a strategy and practice to manage all the elements depicted on the screen. AWS commits to the security of its platform and distributed infrastructure as a service to their customer. While benefiting from their expertise and scale, there is a massive boon for customers. It still means you need to have a plan to secure everything above the line.
+
+#### Categories in Cloud Shared Responsibility Model
+
+There are three categories of responsibilities in the Shared Responsibility Model:
+
+ ![Cloud Shared Responsibility Model](/images/inspec/cloud_security_aws_shared_model.png)
+
+- Inherited controls - Responsibilities that are always the providers.
+
+  The security responsibilities that are always the providers relate to safeguarding the infrastructure itself and access to, patching, and configuration of the physical hosts and the physical network on which the compute instances run, power availability, and the storage and other resources reside. The security tasks extend by protecting end-user data, applications, operating systems, endpoints, and network traffic.
+
+- Customer-specific infrastructure - Responsibilities that are always the customer’s, and
+
+  The security responsibilities that are always the customer’s include managing users and their access privileges (identity and access management), safeguarding cloud accounts from unauthorized access, the encryption and protection of cloud-based data assets, security of the zone data, and managing its security posture compliance.
+
+- Shared controls - Responsibilities that vary depending on the service model.
+
+  Infrastructure as a Service (IaaS), Platform as a Service (PaaS), or Software as a Service (SaaS),  such as cloud email:
```

Comment:
Probably delete this line since it doesn't add anything that isn't in the following paragraphs.
```suggestion
```